ui/component/channelDiscussion/index.js

```diff
@@ -11,6 +12,7 @@ const select = (state, props) => {
 
   return {
     linkedComment: makeSelectCommentForCommentId(linkedCommentId)(state),
+    tags: makeSelectTagsForUri(props.uri)(state),
```

Comment:
This should use
`commentsDisabled: makeSelectTagInClaimOrChannelForUri(props.uri, DISABLE_COMMENTS_TAG)(state),`